Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Changed
 - Use `ProfilerMarker` instead of `Profiler.BeginSample` and `Profiler.EndSample` when appropriate to enable recording of profiling data.
+- Removed aliased values in GamepadButton enum. 
```

Comment:
We do not want to release a major revision due to this change.

Reply:
Should we plan to do so down the line? Perhaps mark the aliased values as depricated (I'm not sure how we go about doing this).

Reply:
We basically shouldn't modify the enum at all, we should aim for a completely editor-based "fix" here. There is no functional issue with the current solution, its only confusing users from UX perspective.

Reply:
It makes sense though to file an internal ticket to consider removing aliases for control enums down the line (in a major future release). I have now filed it.